searx/engines/bt4g.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+ BT4G (Videos, Music, Files)
+"""
+
+from lxml import etree
+import re
+from datetime import datetime
+from urllib.parse import quote
+from searx.utils import get_torrent_size
+
+# about
+about = {
+    "website": 'https://bt4gprx.com',
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": 'XML',
+}
+
+# engine dependent config
+categories = ['files']
+paging = True
+
+# search-url
+url = 'https://bt4gprx.com'
+search_url = url + '/search?q={search_term}&p={pageno}&page=rss'
+
+
+# do search-request
+def request(query, params):
+    params['url'] = search_url.format(search_term=quote(query), pageno=params['pageno'] - 1)
```

Comment:
> Not sure where my issue is but I get always page one in my reader, even I set p=2.

I found my issue: there `params['pageno'] - 1` there is no page number `0` in bt4g's rss  .. so I get for page one and page two in SearXNG the same result / we have to start with `p=1` --> `pageno=params['pageno']` 

@unixfox  I will fix this issue with some other nits I have found .. coming soon.